cmd/ak/common/runtimes.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+import "go.autokitteh.dev/autokitteh/backend/runtimes"
+
+var LocalRuntimes = runtimes.New()
```

Comment:
It's wasteful to create this even when it's not needed, especially when it's used only once so far. Please instantiate `runtimes.New()` directly in `cmd/builds/create.go` instead of this file.

Reply:
ha, done